src/umap/store/SparseStore.h

```diff
@@ -1,4 +1,3 @@
-  
 //////////////////////////////////////////////////////////////////////////////
 // Copyright 2017-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
update to 2021 copyright

Reply:
I've updated the copyrights and pushed the changes.
Thanks!